spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject do
+    User.new(name: 'Tobin', email: 'tobin@gmail.com', password: 'password')
+  end
+  before { subject.save }
+
+  it 'Name should not be nil' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'Email should not be nil' do
+    subject.email = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
[OPTIONAL]
Consider positive and negative scenarios when doing tests. While this is a simple application, doing this in the real world would help catch bugs. What do you expect to happen when negative as well as what do you expect to be output when positive. Consider this for all tests written before merging.

Reply:
Thanks for drawing our attention to this @mcrd25. We will definitely look into that.